wgpu-types/src/lib.rs

```diff
@@ -1309,7 +1309,7 @@ pub enum TextureFormat {
     // Normal 8 bit formats
     /// Red channel only. 8 bit integer per channel. [0, 255] converted to/from float [0, 1] in shader.
     R8Unorm,
-    /// Red channel only. 8 bit integer per channel. [-127, 127] converted to/from float [-1, 1] in shader.
+    /// Red channel only. 8 bit integer per channel. [-128, 127] converted to/from float [-1, 1] in shader.
```

Comment:
I think the old comment is correct. -127 maps to -1.0

Reply:
sorry, i think you are right. i found the specification for vulkan. https://www.khronos.org/registry/vulkan/specs/1.1/html/vkspec.html#fundamentals-fixedfpconv

Reply:
i have removed those of the modification.